src/KeyboardInputModal.scss

```diff
@@ -1,9 +1,7 @@
 @import '_variables';
 
 .KeyboardInputModal__content {
-    overflow: auto;
```

Comment:
Modal content is where we should place the scrollbar I think. Only containers in the modal those should always visible without scrolling should be the header and footer.

Reply:
I had a chat with Simon, and they think it might be worth to render the control outside of the scrollable container. Let's talk about potential options during next meeting.